src/main/java/net/neoforged/neoforge/registries/datamaps/builtin/NeoForgeDataMaps.java

```diff
@@ -73,6 +78,19 @@ public class NeoForgeDataMaps {
     public static final DataMapType<EntityType<?>, MonsterRoomMob> MONSTER_ROOM_MOBS = DataMapType.builder(
             id("monster_room_mobs"), Registries.ENTITY_TYPE, MonsterRoomMob.CODEC).synced(MonsterRoomMob.WEIGHT_CODEC, false).build();
 
+    /**
+     * The {@linkplain Block} data map that replaces {@link WeatheringCopper#NEXT_BY_BLOCK}.
+     * <p>
+     * The location of this data map is {@code neoforge/data_maps/item/oxidizing.json}, and the values are objects with 1 field:
+     * <ul>
+     * <li>{@code after}, a block that object should convert into once it changes oxidizing states</li>
+     * </ul>
+     *
+     * The inverted map of this can be found at {@link DataMapHooks#getInverseOxidizablesMap()}
+     */
```

Comment:
```suggestion
    /**
     * The {@linkplain Block} data map that replaces {@link WeatheringCopper#NEXT_BY_BLOCK}.
     * <p>
     * The location of this data map is {@code neoforge/data_maps/item/oxidizables.json}, and the values are objects with 1 field:
     * <ul>
     * <li>{@code next_oxidized_stage}, a block that the object should convert into once it changes oxidizing states</li>
     * </ul>
     *
     * The inverted map of this can be found at {@link DataMapHooks#getInverseOxidizablesMap()}
     */
```